src/mlpack/methods/time_series_forecast/exponential_smoothing/ses.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file methods/time_series/exponential_smoothing/ses.hpp
+ * @author Aditi Pandey
+ *
+ * Single exponential smoothing.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_EXPONENTIAL_SMOOTHING_SES_HPP
+#define MLPACK_METHODS_EXPONENTIAL_SMOOTHING_SES_HPP
+
+#include <mlpack/prereqs.hpp> 
+#include <armadillo>
+
+namespace mlpack {
+namespace ts /** Single Exponential smoothing method of time series. */ {
+
+/**
+ * Single Exponential Smoothing, SES for short, also called
+ * Simple Exponential Smoothing, is a time series forecasting
+ * method for univariate data without a trend or seasonality.
+ */
+
+// function for calculating mean absolute error
+double calc_mae(const arma::rowvec& errors); 
+// function for calculating mean squared error
+double calc_mse(const arma::rowvec& errors); 
```

Comment:
The name of the function doesn't match its definition in the other file.

Reply:
Yes, I missed those.